src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableConstructor.java

```diff
@@ -0,0 +1,22 @@
+/*
+FinalLocalVariable
+validateEnhancedForLoopVariable = (default)false
+tokens = PARAMETER_DEF
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.coding.finallocalvariable;
+
+public class InputFinalLocalVariableConstructor {
+
+    InputFinalLocalVariableConstructor(int a) { // violation
```

Comment:
All violations must have message.
The only exception is line with two violations

Reply:
done